tests/tools/nnpackage_run/src/nnpackage_run.cc

```diff
@@ -206,6 +224,22 @@ int main(const int argc, char **argv)
       RandomGenerator(session).generate(inputs);
 #endif
 
+    const auto &parallel_inputs = args.getParallelInputs();
+    const auto backend_list = getAvailableParallelBackendList(available_backends);
+    if (parallel_inputs.size() != 0)
+    {
+      // Check whether supported backends is used
+      if (backend_list.size() != 1 || (backend_list.size() != 0 && backend_list.at(0) != "trix"))
```

Comment:
I'm not sure if `nnpackage_run` has a dependency on the specific backend name `trix`.

Reply:
OK, I will move this exception handling into `onert`'s loaders.